src/core/Track.cpp

```diff
@@ -1178,13 +1178,12 @@ TrackContentWidget::~TrackContentWidget()
 
 void TrackContentWidget::updateBackground()
 {
-	const int tactsPerBar = 4;
 	const TrackContainerView * tcv = m_trackView->trackContainerView();
 
-	// Assume even-pixels-per-tact. Makes sense, should be like this anyways
-	int ppt = static_cast<int>( tcv->pixelsPerTact() );
+	// Assume even-pixels-per-bar. Makes sense, should be like this anyways
+	int ppb = static_cast<int>( tcv->pixelsPerBar() );
 
-	int w = ppt * tactsPerBar;
+	int w = ppb * 4;
```

Comment:
Is it actually correct? Couldnt tactsPerBar change depending on the time signature?

Reply:
I don't even get what `tactsPerBar` means. A tact is a bar, so it should always be `1` :smile: But it's `4`, so that really can't be what they meant...?

Reply:
I guess it's about the background color.
![image](https://user-images.githubusercontent.com/12591074/57212035-5c839300-701d-11e9-99ae-9227a44621d8.png)
Here 4 bars form a group. It looks weird with the time signature changed, e.g. 5/3, but it's out of scope.

Reply:
Indeed, it is about the background color.

Reply:
Then can you rename the constant to something like barsPerGroup and leave a comment explaining what it's there for? I feel like the current change makes things harder for someone who wants to adjust these regions in the future.

Reply:
@Spekular 
Done.